jobs/terraform-deploy/terraform-deploy.groovy

```diff
@@ -0,0 +1,61 @@
+job {
+  name 'terraform-deploy'
+  description('Deploy and automatically update Terraform environments upon repository changes')
+  parameters {
+    choiceParam("action", ["apply", "destroy"], "Select if you want to apply changes to environments or destroy them.")
```

Comment:
Personally I like longer lines - they read much easier for me. I believe any decent editor/viewer should be able to do word wrap for you to any desired width you set. I am not sure however how to enable word wrap in github viewer.